src/main/app-paths/app-name/app-name.injectable.ts

```diff
@@ -3,11 +3,19 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 import { getInjectable } from "@ogre-tools/injectable";
-import electronAppInjectable from "../get-electron-app-path/electron-app/electron-app.injectable";
+import packageInfo from "../../../../package.json";
+import isDevelopmentInjectable from "../../../common/vars/is-development.injectable";
 
 const appNameInjectable = getInjectable({
   id: "app-name",
-  instantiate: (di) => di.inject(electronAppInjectable).name,
+
+  instantiate: (di) => {
+    const isDevelopment = di.inject(isDevelopmentInjectable);
+
+    return `${packageInfo.productName}${isDevelopment ? "Dev" : ""}`;
+  },
+
+  causesSideEffects: true,
```

Comment:
Why is this marked as `causesSideEffects`? How is it causing any more side effects then those caused by `isDevelopmentInjectable`?

Reply:
PackageJson. :)

Reply:
Eh I don't think accessing the package json's produce name is within scope for unit tests to need to code around.

Reply:
That doesn't make it free from side-effects still. There is no point not to globally override it. It only makes tests more stable without any downsides.